File: Web/src/main/java/com/itechart/webflux/web/core/model/entity/MeetingUser.java
package com.itechart.webflux.web.core.model.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "meetingUser")
public class MeetingUser implements Entity {

    @Id
    private String id;
    private String userId;
    private String meetingId;

}

File: Web/src/main/java/com/itechart/webflux/web/core/model/entity/MeetingUser.kt
package com.itechart.webflux.web.core.model.entity

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "meetingUser")
class MeetingUser(@Id override var id: String?, var userId: String?, var meetingId: String?) : Entity